Web/Admin.UI/src/app/components/validation-config/validation-categories/edit-validation-category/edit-validation-category.component.ts

```
@@ -0,0 +1,209 @@
+      error: (error) => {
+        console.error('Error loading rules:', error);
+        this.error = 'Failed to load rules';
+        this.isLoading = false;
+      }
+    });
+  }
+
+  onEdit(ruleSet?: IValidationRuleSet): void {
+    let rule = null;
+    
+    if (ruleSet && ruleSet.rules && ruleSet.rules.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)